examples/arm/setup.sh

```diff
@@ -167,7 +167,7 @@ function setup_tosa_reference_model() {
     # errors for the early codebase
     cd "${root_dir}"
     if [[ ! -e reference_model ]]; then
-        git clone https://review.mlplatform.org/tosa/reference_model -b v0.80.0
+        git clone https://review.mlplatform.org/tosa/reference_model -b main
```

Comment:
I understand that the change here is a temporary workaround to unblock this PR just because the new version [`0.90.0d` is not available](https://review.mlplatform.org/plugins/gitiles/tosa/reference_model/). I think the right way for future update will still be the version based (stable) instead attempting to detect and build on new hash from main. So the process would be as simple as:
  1. Make a new version available in https://review.mlplatform.org/plugins/gitiles/tosa/reference_model/
  2. Create a PR to update examples/arms/setup.sh to pick up the new version. In that PR, we should expect to see the docker image is rebuilt using the updated version and `test-arm-reference-delegation` is using the updated version
  
cc: @huydhn @Jerry-Ge

Reply:
Agree, pinning the stable version is definitely the way to go on CI once the version is ready.